packages/bot/src/stores/flyout-store.js

```diff
@@ -155,39 +162,39 @@ export default class FlyoutStore {
                 current_block_index = index;
             }
         });
-        const last_position = Object.keys(xml_list_group).length - 1;
-
-        const adjustIndexOutOfBounds = (current_index, last_index) => {
-            let current_pos = current_index;
-            if (current_pos < 0) {
-                current_pos = last_index;
-            } else if (current_pos > last_index) {
-                current_pos = 0;
-            }
 
-            return current_pos;
+        const getNextBlock = async (xml, currentIndex, direction) => {
```

Comment:
- Please use snake_case for variables (`currentIndex` should be `current_index`)
- You can refactor this function to always return a value (so you can remove the eslint exceptions).

```JavaScript
const getNextBlock = async (xml, currentIndex, direction) => {
    const nextIndex = currentIndex + (direction ? 1 : -1);
    const block_type = Object.keys(xml).find((key, index) => nextIndex === index);

    if (!block_type)  {
        return false;
    }

    try {
        await import(`../scratch/help-content/help-string/${block_type}.json`);
        return block_type;
    } catch (e) {
        return getNextBlock(xml,nextIndex,direction);
    }
};
```

Reply:
done